components/content/Cargo.toml

```diff
@@ -8,6 +8,7 @@ tera = "0.11"
 serde = "1"
 slug = "0.1"
 rayon = "1"
+globset = "0.3.0"
```

Comment:
possible to put it in dev-dependencies as it's only used in tests I think

Reply:
Good spot, changed.